app/views/pregnancies/_patient_information.html.erb

```diff
@@ -9,11 +9,8 @@
 
       <div class="row">
         <div class="info-form-left col-sm-6">
-          <%= f.fields_for pregnancy.patient do |p| %>
-            <%= p.text_field :secondary_person, autocomplete: 'off' %>
-            <%= p.text_field :secondary_phone, autocomplete: 'off' %>
-          <% end %>
-    
+          <strong>First and last name:</strong><br><%= pregnancy.patient.name %><br>
```

Comment:
I think I misinterpreted this in a sleep deprived state -- thought this was moving name and primary phone down into another section, but I see it's not. Gonna pop these out for the time being and slightly edit the labels down in the secondary section.